CHANGELOG.md

```diff
@@ -2,6 +2,41 @@
 
 The release log for BoTorch.
 
+## [0.7.3] - Nov 10, 2022
+
+### Highlights
+* #1454 Fixes a critical bug that affected multi-output `BatchedMultiOutputGPyTorchModel`s that were using a `Normalize` or `InputStandardize` input transform and trained using `fit_gpytorch_model/mll` with `sequential=True` (was the default until 0.7.3). The input transform buffers would be reset after model training, leading to the model being trained on normalized input data but being evaluated on raw inputs. This bug was affecting model fits since the 0.6.5 release.
```

Comment:
```suggestion
* #1454 fixes a critical bug that affected multi-output `BatchedMultiOutputGPyTorchModel`s that were using a `Normalize` or `InputStandardize` input transform and trained using `fit_gpytorch_model/mll` with `sequential=True` (which was the default until 0.7.3). The input transform buffers would be reset after model training, leading to the model being trained on normalized input data but evaluated on raw inputs. This bug had been affecting model fits since the 0.6.5 release.
```